pythonforandroid/recipes/pydantic/__init__.py

```diff
@@ -0,0 +1,12 @@
+from pythonforandroid.recipe import PythonRecipe
+
+
+class PydanticRecipe(PythonRecipe):
+    version = '1.8.2'
+    url = 'https://github.com/samuelcolvin/pydantic.git'
```

Comment:
Use a versioned URL and also avoid git, because it's slower to git clone than to download a zip
```suggestion
    url = 'https://github.com/samuelcolvin/pydantic/archive/refs/tags/v{version}.zip'
```